clang/test/ParserHLSL/hlsl_annotations_on_struct_members.hlsl

```diff
@@ -0,0 +1,30 @@
+// RUN: %clang_cc1 -triple dxil-pc-shadermodel6.0-compute -x hlsl -o - %s -verify
+
+// previously, this test would result in an error shown below on the line that 
+// declares variable a in struct Eg9:
+// error: use of undeclared identifier
+//     'SV_DispatchThreadID'
+// This is because the annotation is parsed as if it was a c++ bit field, and an identifier
+// that represents an integer is expected, but not found.

```

Comment:
No need to explain what the test did before a change in the test itself - that kind of information belongs in the PR description / commit message.